### ISSUE TITLE
[BUG] Issue with disabling enchantability and keeping deadly module still active.

### ISSUE THREAD

**Issue by BastionNtB** [Shadows-of-Fire/Apotheosis#287]
Disabling the enchantment module with the deadly module still active, causes some strange results for the mini boss mods.

Here's a picture
![image](https://user-images.githubusercontent.com/15705306/103447727-f01ec400-4c43-11eb-8b61-61d74f06c008.png)
![image](https://user-images.githubusercontent.com/15705306/103447789-a97d9980-4c44-11eb-9d33-d6ad2cfaf7da.png)

Here's the related mods
Forge 35.1.13
Apotheosis-1.16.3-4.4.1.jar
~ConfigurableCane-1.16.3-2.0.0.jar~ (Removed)
EnchantmentDescriptions-1.16.4-6.0.2.jar
Hwyla-forge-1.10.11-B78_1.16.2.jar
jei-1.16.4-7.6.0.62.jar
~branders-spawnermod-1.16.4-1.7.9.jar~ (Removed)

Can we get the deadly module to work without the enchantment module enabled? Maybe it gives a bunch of vanilla style enchantments instead?

**Comment by Shadows-of-Fire** [comment]
This has nothing to do with the ench module, this is a translation failure, also, still two incompatible mods installed.

Additionaly, disabling modules is equal to removing a mod, and should not be done within a world.

**Comment by BastionNtB** [comment]
Just to clear any confusion. Those incompatible mods are now removed. I recreated the world AGAIN, starting from complete scratch. But the issues still remain.
![image](https://user-images.githubusercontent.com/15705306/103448062-b8664b00-4c48-11eb-972b-8e2f9fd0660b.png)

I would NOT remove a mod such as this, without regenerating the world and come to you with this issue.

So again, the incompatible mods are removed, the world has been regenerated, and the issue is still there.
